src/main/java/me/ruibin/leto/tasklist/TaskList.java

```diff
@@ -0,0 +1,172 @@
+package me.ruibin.leto.tasklist;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import me.ruibin.leto.storage.Storage;
+import me.ruibin.leto.ui.Ui;
+
+/** Class representing the list of tasks in Leto. */
+public class TaskList {
+    private static final ArrayList<Task> list = new ArrayList<>(100);
+
+    public TaskList() {}
```

Comment:
A bit minor but I think Java will fill in this constructor by default so you may not need this line